src/app/actions.ts

```diff
@@ -39,7 +39,7 @@ export async function involvedAction(
 	}
 
 	const sendMailData = sendMail({
-		from: 'MUQA <postmaster@forum.zazelenimo.com>',
+		from: 'MUQA <postmaster@email.muqa.org>',
```

Comment:
Please load the domain part from the environment.

Reply:
Done, thx